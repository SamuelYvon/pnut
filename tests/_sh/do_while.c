#include <stdio.h>

int factorial(int num) {
  int ret = 1;
  
  do {
    ret *= num;
    num--;
  } while(num > 0);

  return ret;
}

void main() {
  printf("Factorial of 5: %d\n", factorial(5));
  printf("Factorial of 10: %d\n", factorial(10));
}
