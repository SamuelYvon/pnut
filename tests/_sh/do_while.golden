Factorial of 5: 120
Factorial of 10: 3628800
